### DISCUSSION TITLE
when pushing prisma schema to supabase an error Error: P1001: Can't reach database server at `db.xxxxx.supabase.co`:`xxxx`

### DISCUSSION THREAD

**Discussion by menna13helal** [supabase/supabase#14975] [Questions]
I am using prisma "prisma": "^4.8.1", and "@prisma/client": "^4.8.1",
I added my models in prisma schema

when I try to push prisma schema to supabase
using npx prisma db push

an error occured 
Error: P1001: Can't reach database server at `db.xxxxx.supabase.co`:`xxxx`

Notes:
it was working fine yesterday, suddenly this error kept appearing
I tried creating new project, organization ... nothing is working
also, I tried to restart my project ... again nothing is working

PS: I saw this question before, and I tried the recommended solutions, but nothing is working

can anyone help please ASAP?
Thanks in advance

**Comment by menna13helal** [comment]
the issue is solved after contacting supabase support

**Reply by andmap** [reply]
so how to fix the issue, we had this today, was working fine, and did npx prisma db push.
